plaid/institutions.go

```diff
@@ -36,10 +40,17 @@ type ItemLogins struct {
 	Breakdown        InstitutionStatusBreakdown `json:"breakdown"`
 }
 
+type InstitutionStatusFields struct {
```

Comment:
We could replace the `ItemLogins` struct with `InstitutionStatusFields` for simplicity.

Reply:
We didn't want to modify the type of an existing field because it would create a breaking change.